src/transactions/InvokeHostFunctionOpFrame.cpp

```diff
@@ -320,7 +362,15 @@ InvokeHostFunctionOpFrame::doApply(Application& app, AbstractLedgerTxn& ltx)
     metrics.mMemByte = out.mem_bytes;
     if (!metrics.mSuccess)
     {
-        innerResult().code(INVOKE_HOST_FUNCTION_TRAPPED);
+        if (resources.instructions < out.cpu_insns ||
```

Comment:
This is nice, but just wanted to make sure: is it guaranteed that the output value will exceed the resources (i.e. that we increment the counter before panic)?

Reply:
Yes, the counter is always incremented before erroring.

Reply:
Sounds good, thanks for the confirmation.